proposals/0282-record-dot-syntax.rst

```diff
@@ -107,13 +131,28 @@ thus, ``f r.a.b`` parses as ``f (r.a.b)``.
 Expression     Interpretation
 ============== ===================
 ``f r.x``      means ``f (r.x)``
-``f M.n.x``    means ``f (M.n.x)``
-``f M.N.x``    means ``f (M.N.x)``
 ``f r .x``     is illegal
 ``f (g r).x``  ``f ((g r).x)``
 ``f (g r) .x`` is illegal
+``f M.n.x``    means ``f (M.n.x)`` (that is, ``f (getField @"x" M.n)``)
 ============== ===================
 
+Disambiguation of ``M.N.x`` is treated as follows.
+
+============== ===================
+Expression     Interpretation
+============== ===================
+``f M.N.x``    means ``f (M.N.x)`` (``M.N.x`` is a qualified name, not a record field selection)
+============== ===================
+
+Note: ``M.N.x`` looks ambiguous. It could mean:
```

Comment:
Put this first, and not as a "Note".  It's simply a disambiguation rule.

Reply:
- Done.